src/Categories.js

```diff
@@ -1,3 +1,7 @@
+import { useDispatch } from 'react-redux';
+import { checkCargory } from './redux/categories/categories';
+
 export default function Categories() {
-  return <><button type="button">Check status</button></>;
+  const dispatch = useDispatch();
+  return <><button type="button" onClick={dispatch(checkCargory())}>Check status</button></>;
```

Comment:
- [ ] The button "Check Status" should show the message "Under Construction"

Reply:
check the documentation only dispatching the action is required